buildenv/jenkins/variables/defaults.yml

```diff
@@ -167,7 +172,8 @@ boot_jdk_default:
       8: '8'
       11: '11'
       16: '16'
-      next: '16'
+      17: '16'
+      next: '17'
```

Comment:
next needs to stay at 16 until 17 is released. It won't require 17 until after release.

Reply:
Updated. Just trying to push the repo up in order to try a build.

Reply:
Do you want me to keep this un-merged until the repo is ready and we can do a jdk17 PR?